### PR TITLE
[IMP] l10n_jo: Updated taxes and tax groups

### PR DESCRIPTION
**Pull request by hsal-odoo**
Updated the tax names, descriptions, and invoice labels to be more clear.
Added an additional tax group for exempt.

task-4196993

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_jo/data/template/account.tax-jo_standard.csv [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
 "","","","","","","","","tax","invoice","+2b (Tax)","jo_account_200901",""
 "","","","","","","","","base","refund","-2b (Base)","",""
 "","","","","","","","","tax","refund","-2b (Tax)","jo_account_200901",""
-"jo_standard_purchase_4","4%","purchase","4.0","percent","4% (2c)","4%","jo_tax_group_4","base","invoice","+2c (Base)","",""
+"jo_standard_purchase_4","4%","purchase","4.0","percent","","4%","jo_tax_group_4","base","invoice","+2c (Base)","",""
 "","","","","","","","","tax","invoice","+2c (Tax)","jo_account_200901",""
 "","","","","","","","","base","refund","-2c (Base)","",""
 "","","","","","","","","tax","refund","-2c (Tax)","jo_account_200901",""
-"jo_standard_purchase_other","0% O","purchase","0.0","percent","0% (2d) Other Tax","0%","jo_tax_group_0","base","invoice","+2d (Base)","","2d. ضريبة المشتريات الأخرى"
+"jo_standard_purchase_other","0% O","purchase","0.0","percent","Other","0% Other","jo_tax_group_0","base","invoice","+2d (Base)","","2d. ضريبة المشتريات الأخرى"
```
Comment (malb-odoo) [inline addons/l10n_jo/data/template/account.tax-jo_standard.csv:14]: i know it's not you but the translation seems a bit weird, could we have confirmation on that ? also maybe we need to translation the description then

**Review by malb-odoo — COMMENTED** [review]
Thankkkss just a question for the translation otherwise LGTM

File: addons/l10n_jo/data/template/account.tax-jo_standard.csv (hunk 1 above)
Comment (hsal-odoo) [reply addons/l10n_jo/data/template/account.tax-jo_standard.csv]: Yeah seems like the translation were already meant to be for the descriptions. I fixed it now in a separate commit, should I merge the 2 commits into 1 or what do u think?

Comment (malb-odoo) [reply addons/l10n_jo/data/template/account.tax-jo_standard.csv]: One commit is ok i think :smile:

Comment (hsal-odoo) [reply addons/l10n_jo/data/template/account.tax-jo_standard.csv]: Also, wouldn't removing names translations from stable be an issue?

Comment (malb-odoo) [reply addons/l10n_jo/data/template/account.tax-jo_standard.csv]: For csv like here i think it's ok :smile:

**Review by malb-odoo — COMMENTED** [review]
LGTM :smile: Thanks for that ! 
cc: @william-andre not sure it's need a post_migrate script try_loading when changing the tax group on a tax ? Or we don't care and the fix will be applied for new db only ?

**Review by malb-odoo — APPROVED** [review]

**Review by william-andre — APPROVED** [review]
Yeah, no need for an upgrade script
@robodoo r+

**Pull request closed without merging**